build.gradle

```diff
@@ -118,6 +118,7 @@ subprojects {
 
         testCompile 'org.spockframework:spock-core'
         testCompile 'org.spockframework:spock-spring'
+        testCompile 'cglib:cglib-nodep:2.2.2'
```

Comment:
If this dependency isn't available as part of the spring platform bom please move it to gradle.properties so we can try to keep it consistent across project if we end up using it in multiple places